### PR TITLE
Add wrappers for tracing-aware ExecutorServices/Callables/Runnables

### PR DESCRIPTION
**Pull request by uschi2000**

### CODE REVIEW

File: tracing/src/main/java/com/palantir/remoting1/tracing/Traces.java [hunk 1]
```diff
@@ -16,13 +16,103 @@
 
 package com.palantir.remoting1.tracing;
 
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.ThreadLocalRandom;
 
+/** Utility methods for making {@link ExecutorService} and {@link Runnable} instances tracing-aware. */
 public final class Traces {
```
Comment (markelliot) [inline tracing/src/main/java/com/palantir/remoting1/tracing/Traces.java]: should we call this Tracers?

**Review by markelliot — APPROVED** [review]

File: tracing/src/main/java/com/palantir/remoting1/tracing/Traces.java (hunk 1 above)
Comment (uschi2000) [reply tracing/src/main/java/com/palantir/remoting1/tracing/Traces.java]: sure

**Comment by uschi2000** [comment]
renamed class and rebased

**Comment by ashrayjain** [comment]
@uschi2000 why was this copied from Guava? Do we still need it?
Edit: nvm, just saw its package private in Guava.

**Pull request merged**